src/main/java/com/sparta/outsourcing/domain/menu/repository/MenuRepository.java

```diff
@@ -1,10 +1,13 @@
 package com.sparta.outsourcing.domain.menu.repository;
 
 import com.sparta.outsourcing.domain.menu.entity.Menu;
+import jakarta.validation.constraints.NotBlank;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import java.util.List;
 
 public interface MenuRepository extends JpaRepository<Menu, Long> {
     List<Menu> findAllByStoreStoreId(Long storeId);
+
+    Menu findByName(@NotBlank(message = "메뉴 이름을 입력해주세요.") String name);
```

Comment:
```suggestion
    Optional<Menu> findByStoreStoreIdAndName(Long storeId, String name);
```

Reply:
```suggestion
    Optional<Menu> findByStoreStoreIdAndNameAndDeletedIsTrue(Long storeId, String name);
    Optional<Menu> findByStoreStoreIdAndNameAndDeletedIsFalse(Long storeId, String name);
```
해당 가게의 메뉴만 확인하기에 storeId를 넣었습니다.
deleted는 여기서 확인해도 되고, service에서 확인하셔도 됩니다.